model/riscv_platform.sail

```diff
@@ -24,6 +24,12 @@ val elf_entry = pure {
   c: "elf_entry"
 } : unit -> int
 
+function readable_addr_bits (addr : physaddrbits) -> string = {
```

Comment:
```suggestion
function PhysaddrStr(addr : physaddr) -> string = {
```
Perhaps change the signature to ensure it can't be accidentally called on a bitvector, and the name to match the existing `RegStr` and `FRegStr`.   `PAddrStr` could also work as a shorter name.